gamenews

```diff
@@ -1,4 +1,4 @@
-2023-09-27 10-00-00:
-	Title: Playtest 20270927
+2023-10-10 18-00-00:
+	Title: Release 20231010
 	Author: PunkPun
-	Content: Two months went by, and bam! Another playtest! In an effort of pushing out a swift release we highly encourage everybody to chip in and give the playtest a spin.\n\nSome of the more notable changes in playtest include:\n • Return of darker player colors\n • The ability to vote to kick players\n • New Red Alert and Tiberium Dawn missions\n • Assets installation via Steam or the EA app\n • Red Alert balance changes\n • Reworked sprite sequence loading\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on the forum, Discord, or GitHub.
+	Content: We are pleased to finally announce Release 20231010!\n\nIt restores the ability to select darker player colors, adds vote-kick, contains new Red Alert and Tiberium Dawn missions, adds the ability to install classic assets from Steam and the EA app, and a bunch more!\n\nYou can find the full changelog on the website, this time around we also are highlighting many community mods and tournaments. To checkout the newspost and grab the latest and greatest installment of OpenRA head over to www.openra.net
```

Comment:
Maybe `adds vote-kicking` or even `adds the ability to vote-kick` which will then also solve your spacing issue with `greatest`?

Reply:
it would not work as it's a \n\n away

Reply:
<img width="444" alt="Screenshot 2023-10-10 at 15 33 56" src="https://github.com/OpenRA/OpenRAWebsiteV3/assets/37534529/fcd291f3-309d-4bf0-94a8-db0531875887">

managed to get it to look semi-normal by adding `upcoming` instead